src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationlocation/InputAnnotationLocationMultiple.java

```diff
@@ -0,0 +1,36 @@
+package com.puppycrawl.tools.checkstyle.checks.annotation.annotationlocation;
+
+import java.lang.annotation.Repeatable;
+
+/**
+ * This test-input is intended to be checked using following configuration:
+ *
+ * allowSamelineSingleParameterlessAnnotation = false
+ * allowSamelineParameterizedAnnotation = false
+ * allowSamelineMultipleAnnotations = true
+ */
```

Comment:
Can we have some warnings in this file ?

Reply:
No. with `allowSamelineMultipleAnnotations = true` both `allowSamelineSingleParameterlessAnnotation` and `allowSamelineParameterizedAnnotation` has no effect. Actually, this means that any combination of annotations is allowed.

However, it is possible to catch a violation by using annotations on different lines.
This one: https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java#L292
But this is a different case and should be covered by another set of tests.